src/main/java/com/worksap/fig/lang/Seq.java

```diff
@@ -389,6 +390,32 @@ default int findLastIndex(Predicate<T> condition) {
     }
 
     /**
+     * Create a new seq with size null elements
+     */
+    static <T> Seq<T> newSeq(int size) {
+        return new SeqImpl<>(size);
+    }
+
+    /**
+     * Create a new seq with size defaultValue elements
+     */
+    static <T> Seq<T> newSeq(int size, T defaultValue) {
+        return new SeqImpl<>(size, defaultValue);
+    }
+
+    /**
+     * Create a new seq with size elements, each of elements is generated by a {@link Function}.
+     */
```

Comment:
Take care of English, write good documentation. For example:

What do you mean by "size elements"?

"each of elements is generated by a {@link Function}". Users will ask "what is the input of this function?", index or anything else?

The original Ruby documentation:

> In the last form, an array of the given size is created. Each element in this array is created by passing the element’s index to the given block and storing the return value.